### ISSUE TITLE
Restart error

### ISSUE THREAD

**Issue by plovely** [underworldcode/UWGeodynamics#184]
Hi Romain, Julian, et al.,

There appears to be a but in UWGeo v2.9.5 when loading passive tracers in a model restart.  

The same input file works fine in UW2, v2.9.1b. 

Let me know if you need additional info.

Thanks,
Peter

![image](https://user-images.githubusercontent.com/30320218/81092528-54776100-8ec6-11ea-8a20-00d890c2ed3a.png)

**Comment by rbeucher** [comment]
Hi Peter,

Can you send me the h5 file of the step you are trying to reload. And maybe the lines where you create the passive tracers.

Thanks

**Comment by plovely** [comment]
Hi Romain, 

I just sent the requested files to your unimelb.edu.au email.

Here is the python script with which I'm creating the tracers:

```
import numpy as np
x = np.linspace(GEO.nd(Model.minCoord[0]), GEO.nd(Model.maxCoord[0]), 1000)
topo = GEO.nd(uppercrust.top)
interface2 = GEO.nd(lowercrust.top)
interface3 = GEO.nd(mantleLithosphere.top)

topo = Model.add_passive_tracers(name="Topo", vertices=[x, topo])
interface2 = Model.add_passive_tracers(name="Interface2", vertices=[x, interface2])
interface3 = Model.add_passive_tracers(name="Interface3", vertices=[x, interface3])
```

**Comment by rbeucher** [comment]
Can you send it to the ANU one, I don't have access to Unimelb anymore.
romain.beucher at anu.edu.au

**Comment by plovely** [comment]
Done

**Comment by rbeucher** [comment]
I am puzzled... I can't reproduce the bug and looking at the code I don't understand how this would happen. 

Have you changed the script between the first and second run.

Looks like the particle numbers has changed

**Comment by plovely** [comment]
Particles are flowing out the side boundaries,  so the number decreases.
I can send the full model tomorrow.

**Comment by rbeucher** [comment]
This has been fixed in 2.9.6. Will be included in 2.10
